### PR TITLE
Feat/add in experiment

### PR DESCRIPTION
**Pull request by tkhmy**
## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: src/external_signage/src/external_signage/external_signage_core.py [hunk 1]
```diff
@@ -137,14 +160,28 @@ def send_data(self, display_key, data_key):
         sender.send(data, ack_query_ack, ack_data_chunk)
 
     def trigger_external_signage(self, request, response):
-        if not self._external_signage_available:
-            return response
+        if request.data:
```
Comment (yabuta) [inline src/external_signage/src/external_signage/external_signage_core.py]: すみません、ここにself._settings["in_experiment"] == Trueなら表示を変えない条件を入れてください
実験中の場合は自動走行中でもずっと「実証実験中」の表示にしたいです

File: src/external_signage/src/external_signage/external_signage_core.py [hunk 2]
```diff
@@ -114,15 +120,32 @@ def __init__(self, node):
             "back": self._load_display_data(self.protocol.back, package_path),
             "side": self._load_display_data(self.protocol.side, package_path),
         }
+        self._settings_file = "/home/" + os.environ.get("USER") + "/settings.json"
```
Comment (yabuta) [inline src/external_signage/src/external_signage/external_signage_core.py:139]: 今回ではなくてもいいかもですが設定ファイルを置く場所は.envなどで指定できるようにして用途によっておく場所を変更できるようにできると良いかもしれませんね

File: src/external_signage/src/external_signage/external_signage_core.py (hunk 1 above)
Comment (tkhmy) [reply src/external_signage/src/external_signage/external_signage_core.py]: [ed23869](https://github.com/tier4/AutowareSignage/pull/103/commits/ed23869a57d929676c53d1e2bfd2332863d65dd6)で修正しました。

File: src/external_signage/src/external_signage/external_signage_core.py (hunk 2 above)
Comment (tkhmy) [reply src/external_signage/src/external_signage/external_signage_core.py:139]: なるほど、そうですね。
ansibleで追加してもいいと思います。

**Review by yabuta — APPROVED** [review]

**Pull request merged**